### ISSUE TITLE
Kured 4.0.2 chart fails with ImagePullBackOff

### ISSUE THREAD

**Issue by BassOfLion** [kubereboot/charts#8]
It look like there is some issues with kured 4.0.2 chart. Our kured pods fail with ImagePullBackOff status. When you describe pod the following error message appears:

`Failed to pull image "ghcr.io/kubereboot/kured:1.10.2": rpc error: code = Unknown desc = failed to pull and unpack image "ghcr.io/kubereboot/kured:1.10.2": failed to resolve reference "ghcr.io/kubereboot/kured:1.10.2": failed to do request: Head "https://ghcr.io/v2/kubereboot/kured/manifests/1.10.2": EOF`

Any thoughts? 

I managed to pull the image with docker pull command locally. and I can pull the image. Reverting back to chart v4.0.1 for now.

**Comment by ckotzbauer** [comment]
This this a persistent issue? I can pull the image without problems on my local machine.

**Comment by BassOfLion** [comment]
Yes, I have tried multiple times to update the chart and roll it out to our k8s. I get  the same error as above each time. I can too pull the image manually on my local machine.

**Comment by BassOfLion** [comment]
Well, it turns out that we had to open up our network for g**h**cr container registry since v4.0.2 introduces new one... 

gcr and ghcr are not the same thing. Lesson learned. My bad. Closing the issue...
